src/test/ui/functions-closures/fn-help-with-err.rs

```diff
@@ -1,16 +1,28 @@
 // This test case checks the behavior of typeck::check::method::suggest::is_fn on Ty::Error.
+
+struct Foo;
```

Comment:
Had to fix this test since we no longer say "perhaps you wish to call it" on a method that doesn't exist.